<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>menu2</class>
 <widget class="QWidget" name="menu2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>605</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>605</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>605</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>605</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="fondo.qrc">:/Imagenes/fondo.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="save">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>540</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Dead Kansas</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>check</string>
   </property>
  </widget>
  <widget class="QPushButton" name="return_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>530</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Dead Kansas</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>return</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>110</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Dead Kansas</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Answer the questions</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>180</y>
     <width>451</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Dead Kansas</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Â¿personaje principal de legend of zelda:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>390</y>
     <width>431</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Dead Kansas</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>amarillo y come fantasmas:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>270</y>
     <width>461</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Dead Kansas</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>nombre del protagonista de god of war:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pregunta1">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>230</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Dead Kansas</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="pregunta2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>330</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Dead Kansas</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="pregunta3">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>450</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Dead Kansas</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="fondo.qrc"/>
 </resources>
 <connections/>
</ui>
